bin/bundle

```diff
@@ -1,3 +1,3 @@
-#!/usr/bin/env ruby2.1
```

Comment:
Hmm. I'm not sure if this is the right place to ensure that people use the right ruby. :confused: